### PR TITLE
Update choosing-core-framework-server.md

### PR DESCRIPTION
**Pull request by doggy8088**

### CODE REVIEW

**Comment by v-radelg** [comment]
@doggy8088  您好，我們仍然在討論您的建議，會儘速於稍後做出結論。  
感謝您的支持， 
Microsoft C&E 當地語系化小組敬上

**Comment by v-radelg** [comment]
@doggy8088 您好，感謝您的意見反應及建議！   

感謝建議，惟建議譯法實含文法錯誤，原譯正確。

Microsoft C&E 當地語系化小組敬上

**Comment by doggy8088** [comment]
@v-radelg 請問你可以告訴我哪裡**文法錯誤**嗎？

**Comment by v-radelg** [comment]
@doggy8088 
我們會聯繫我們的語言版主為您提供進一步的解釋
Microsoft C&E 當地語系化小組敬上

**Comment by doggy8088** [comment]
@v-radelg Do you have someone who lives in Taiwan or speaking in Chinese and writing in Traditional Chinese? 

I have to say that your original translation is not optimal and quite weird for Traditional Chinese.  For example, the term `scalable` is not usual to translate to `可調整的`.  

Also, for your original translation `您需要依應用程式讓 .NET 版本並存。`.   It's so hard to read and understand.  The translation document is serving LOCAL people.  Easy to read is way to important I think.

Believe me, I'm good than you in Traditional Chinese no matter for writing or reading.  I'm also a book author and speaker and MVP for 10 years.  I'm so confident on this.

**Comment by v-radelg** [comment]
@doggy8088, Dear Will, we truly appreciate your feedback and further comments. We understand you would like more detailed information about our resolution and we have already contacted our Language Moderator to provide you with an extensive answer. Also regarding your first question, indeed our language moderators (who oversee PRs) are native speakers and certified translators, experienced in MS products and services. We should soon have an answer to your new feedback and we will be more than glad to implement the suggested changes if approved 😊
Thanks again for your cooperation and understanding.

**Comment by gosali** [comment]
@doggy8088 - Hello Will. 

We are glad to share with you that we have received answer from our zh-TW Language Moderator with further details. LM has also agreed on one of your suggestions, so we will work on implementing the changes as soon as possible.😊

Please see LMs comments below:
1. The adjective 'scalable' here is derived from the verb 'scale'. There is more than one translation pattern of the verb 'scale', considering this verb is always to be followed by a specific set of particles − scale in, scale out, scale up, scale down. As one of the common translation, the CHT verb 延展 only refers to the part of "scale out". It cannot be used for the rest of the meanings/uses at once. Given that we have to take care of the possibility of 'scale in', 'scale up' and 'scale down', we must choose the more generic 調整 as its root instead of 延展. 
Simply put, considering the translation pattern of scalable here has to take care of all of the possible meaning listed above, we must choose a translation more generic than 延展. Thanks for your understanding.

2. The suggested translation is indeed better. I have changed my mind and accepted it.

If you have any questions or concerns, please let us know and make sure to tag our account so we don’t miss it 😊

Thanks for your support,
Microsoft C&E Localization Team

**Comment by doggy8088** [comment]
Thanks for your clarification.  Now I know more about the meaning of '**scalable**'.  Thank again! 👍
